.gitignore

```diff
@@ -0,0 +1,5 @@
+ditto/
+files/
+index.js
```

Comment:
Personally, I found it unexpected that `npm start` runs the build output and doesn't build on the fly and I would change that, e.g. use a `build/` dir for the building, put that on .gitignore, and then change `npm start` to use `ts-node` or similar.